ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,6 +143,79 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        _buckets: {
+          type: Array,
+          value: () => [],
+          computed: '_setBuckets(buckets)',
+        },
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        _showAll: {
+          type: Boolean,
+          value: false,
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        /**
+         * If collapsible, the default visible items.
+         */
+        visibleItems: {
+          type: Number,
+          value: 8,
+        },
+      };
+    }
+
+    _isCollapsible(buckets) {
+      if (this.collapsible && buckets.length) {
+        return true;
+      }
+    }
+
+    _setBuckets(buckets) {
```

Comment:
If I'm seeing it correctly, we may not need `_buckets` and `_visibleBuckets`.
`this._buckets` will never be different than `this.buckets`, right?